integration_tests/suite/base/test_devices.py

```diff
@@ -123,15 +123,15 @@ def test_search(device, hidden):
     }
 
     for field, term in searches.items():
-        yield check_search, url, device, hidden, field, term
+        yield check_search, url, device, hidden, field, term, 'autoprov'
 
 
-def check_search(url, device, hidden, field, term):
+def check_search(url, device, hidden, field, term, autoprov):
```

Comment:
I would greatly prefer that the autoprov-related tests be separate from the non-autoprov tests :smiley: otherwise that would only test for the autoprov case

Reply:
Both tests now done, for non-autoprov, and for autoprov devices :)
Thank you again !